specs/commandline/plugin.md

```diff
@@ -41,51 +42,87 @@ Aliases:
 ### notation plugin install
 
 ```text
-Installs a plugin
+Install a plugin
 
 Usage:
-  notation plugin install [flags] <plugin path>
+  notation plugin install [flags] <plugin location>
 
 Flags:
   -h, --help                    help for install
   -f, --force                   force the installation of a plugin
-
+      --checksum                verify the checksum digest of a plugin
+      --source string           the location of plugin installation file, options: "file", "url","registry" (default "file")                  
 Aliases:
   install, add
 ```
 
-### notation plugin remove
+### notation plugin uninstall
 
 ```text
-Removes a plugin
+Uninstall a plugin
 
 Usage:
-  notation plugin remove [flags] <plugin name>
+  notation plugin uninstall [flags] <plugin name>
 
 Flags:
   -h, --help          help for remove
 
 Aliases:
-  remove, rm, uninstall, delete
+  remove, rm, uninstall
```

Comment:
The `notation plugin uninstall` command needs to be script friendly as well. Thus, we still need a `--yes` flag to `not prompt for confirmation`.